x-pack/plugins/cases/public/components/create/form_context.test.tsx

```diff
@@ -100,29 +100,39 @@ const defaultPostPushToService = {
   pushCaseToExternalService,
 };
 
-const fillFormReactTestingLib = async (renderResult: RenderResult) => {
+const sampleDataWithoutTags = {
+  ...sampleData,
+  tags: [],
+};
+
+const fillFormReactTestingLib = async (renderResult: RenderResult, withTags = false) => {
   const titleInput = within(renderResult.getByTestId('caseTitle')).getByTestId('input');
 
-  userEvent.type(titleInput, sampleData.title);
+  userEvent.paste(titleInput, sampleDataWithoutTags.title);
 
-  const descriptionInput = renderResult.container.querySelector(
-    `[data-test-subj="caseDescription"] textarea`
+  const descriptionInput = within(renderResult.getByTestId('caseDescription')).getByTestId(
+    'euiMarkdownEditorTextArea'
   );
 
-  if (descriptionInput) {
-    userEvent.type(descriptionInput, sampleData.description);
-  }
+  userEvent.paste(descriptionInput, sampleDataWithoutTags.description);
 
-  const caseTags = renderResult.getByTestId('caseTags');
+  if (withTags) {
+    const caseTags = renderResult.getByTestId('caseTags');
 
-  for (let i = 0; i < sampleTags.length; i++) {
-    const tagsInput = await within(caseTags).findByTestId('comboBoxInput');
-    userEvent.type(tagsInput, `${sampleTags[i]}{enter}`);
+    for (let i = 0; i < sampleTags.length; i++) {
```

Comment:
nit: do we need a counter in the for-loop? Could we switch this to a for-of-loop:

```
for (const tag of sampleTags) {
  ...
}
```

Reply:
Oh sorry, I didn't look at the previous version 🤦‍♂️ that's the way it was before. Ignore me.

Reply:
No worries! It is a good opportunity to improve it